src/main/webapp/src/components/CourseDetails.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div>
+    <div>
+      <h1>{{ course.title }}</h1>
+    </div>
+    <div>
+      <div class="detailThumbnail">
+        <b-img left thumbnail class="mb-3 mr-3" :src="cardimage(course)"></b-img>
+      </div>
+      <b-row class="mb-3" cols="1" cols-md="2">
+        <b-col v-if="course.courseType">
+          <div class="label">Typ:</div>{{ course.courseType=='EXTERNAL' ? 'Externe' : 'Interne' }}
+        </b-col>
+        <b-col v-if="course.location">
+          <div class="label">Ort:</div>{{ course.location=='REMOTE' ? 'Remoteveranstaltung' : 'Präsenzveranstaltung' }}
+        </b-col>
+        <b-col v-if="course.startDate">
+          <div class="label">Start:</div>{{ moment(course.startDate).format('llll') }}
+        </b-col>
+        <b-col v-if="course.endDate">
+          <div class="label">Ende:</div>{{ moment(course.endDate).format('llll') }}
+        </b-col>
+        <b-col v-if="course.address">
+          <div class="label">Adresse:</div>{{ course.address }}
+        </b-col>
+        <b-col v-if="course.organizer">
+          <div class="label">Organisator:</div>{{ course.organizer }}
+        </b-col>
+        <b-col v-if="course.trainer">
+          <div class="label">Trainer:</div>{{ course.trainer }}
+        </b-col>
+        <b-col v-if="course.link">
+          <div class="label">Link:</div><div class="abbreviation"><a :href="course.link">{{ course.link }}</a></div>
+        </b-col>
+      </b-row>
+      <p>{{ course.targetAudience }}</p>
+    </div>
+  </div>
+</template>
+
+<script>
+import axios from 'axios';
+import coffeeImg from '../assets/coffee.jpg';
+import signsImg from '../assets/signs.jpg';
+import moment from 'moment/min/moment-with-locales';
+
+export default {
+  name: "CourseDetails",
+  data: function () {
+    return {
+      course: {}
+    }
+  },
+  methods: {
+    cardimage: function (course) {
+      switch (course.courseType) {
+        case "EXTERNAL":
+          return signsImg;
+        case "INTERNAL":
+        default:
+          return coffeeImg;
+      }
+    }
+  },
+  computed: {
+    moment () {
+      var locale = window.navigator.userLanguage || window.navigator.language;
+      moment.locale(locale);
+      return moment;
+    }
+  },
+  mounted: function () {  
+    // TODO change hostname later
+    axios.get('http://localhost:8080/courses/2')
```

Comment:
the id parameter has to be inserted into the component via prop

Reply:
done